input/docs/handbook/commands/binding-commands.md

```diff
@@ -67,3 +59,5 @@ this.BindCommand(
     x => x.myControl,
     x => x.SomeProperty);
 ```
+**Platform Specific **
+WPF: Arguments passed through CommandParameter in the view are automatically bound to `TInput` in `ReactiveCommand<Input, Unit>`.
```

Comment:
This one isn't WPF specific, also happens on UWP is another platform I can think of.

Reply:
Also might be worthwhile doing more of an example of this one, and also how the preferred approach due to platform compatibility is probably not to use that functionality but instead prefer View/VM binding of the parameters.

Reply:
Also appears as if Xamarin Forms also support the CommandParameter approach https://forums.xamarin.com/discussion/85338/how-to-pass-value-from-binding-in-the-commandparameter-from-a-button

Reply:
Yes, CommandParameter is supported in Xamarin Forms as well, IIRC.

For views generated through viewmodels, we could do VM Binding but for static design, the ViewModel binding might be an unneeded complexity (Introducing a VM property).

Reply:
Potentially, we just do recommendations and leave it up to the user to decide based on their use case, so you can mention those two scenarios.